sections/home/WhatsHappening.jsx

```diff
@@ -0,0 +1,88 @@
+import styled from "styled-components";
+
+import PumpDigest from "../../public/home/pump-digest.png";
+import ImpactReport from "../../public/home/impact-report.png";
+import { CardCarousel, Text } from "../../components";
+import { baseTheme } from "../../theme";
+import { media } from "../../utils";
+
+export const WhatsHappening = ({ ...props }) => (
+  <Wrapper {...props}>
+    <Title size={baseTheme.size.h1} bold="true">
+      What&apos;s Happening?
+    </Title>
+    <CarouselWrapper>
+      <CardCarousel
+        slides={[
+          {
+            title: "PuMP Digest",
+            description:
+              "Check out our blog to explore our articles on research, student life, post-secondary education, careers in medicine, and more!",
+            thumbnail: PumpDigest,
+          },
+          {
+            title: "Impact Report",
+            description:
+              "We are heading into our third year of operations! Read about our projects from the 2020-21 fiscal year in our first ever impact report.",
+            thumbnail: ImpactReport,
+          },
+          {
+            title: "PowerUp Mentorship",
+            description:
+              "Explore articles on research, student life, PuMP has partnered up with The STEM Fellowship to launch our mentorship program, PowerUp!",
+            thumbnail: PumpDigest,
+          },
+          {
+            title: "PuMP Digest 2",
+            description:
+              "Check out our blog to explore our articles on research, student life, post-secondary education, careers in medicine, and more!",
+            thumbnail: PumpDigest,
+          },
+          {
+            title: "Impact Report 2",
+            description:
+              "We are heading into our third year of operations! Read about our projects from the 2020-21 fiscal year in our first ever impact report.",
+            thumbnail: ImpactReport,
+          },
+        ]}
+      />
+    </CarouselWrapper>
+  </Wrapper>
+);
+
+const Wrapper = styled.div`
+  margin: 0 5vw 5vh 5vw;
+`;
+const CarouselWrapper = styled.div`
+  margin: auto;
+  ${media(
+    1200,
+    `
+            width: 70%;
+        `
```

Comment:
(for 1. from the review comment, cap the width size) 
- I *think* this is where carousel width is set 
  - set a constant width after some width
  - eg. if max 3 slides per view, then after 1400 (or other), the carousel will stay the same width and avoid really far apart cards